<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DishDialog</class>
 <widget class="QDialog" name="DishDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>菜品详情</string>
  </property>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>311</width>
     <height>161</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>picname</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_desc">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>492</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>描述</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_abort1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>273</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_confirm">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>273</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_jia">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>51</width>
     <height>43</height>
    </rect>
   </property>
   <property name="text">
    <string>价  格</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_price">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 75 20pt &quot;Adobe Arabic&quot;;</string>
   </property>
   <property name="text">
    <string>123</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_huiyuan">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>会员价</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_vip_price">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 75 20pt &quot;Adobe Arabic&quot;;</string>
   </property>
   <property name="text">
    <string>121</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_numb">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>265</y>
     <width>41</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>数量</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_sub">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>272</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_num">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>275</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_add">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>272</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_n_price">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>264</y>
     <width>51</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>123</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>85</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>￥</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>125</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>￥</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
